lambdas/src/apis/collections/controllers/wearables.ts

```diff
@@ -72,7 +85,7 @@ export async function getWearablesEndpoint(
   offChainManager: OffChainWearablesManager,
   req: Request,
   res: Response
-) {
+): Promise<unknown> {
```

Comment:
Shouldn't this be something like: `{wearables: Wearable[], filters: Filters, pagination: Pagination}`?

Reply:
In this case that's what the response will send, but not what the method returns.

Reply:
what about `any` then?

Reply:
it doesn't work when result is void (no return)